salalib/mapconverter.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "salalib/axialmap.h"
+#include "salalib/shapemap.h"
+#include "salalib/spacepixfile.h"
+#include "genlib/comm.h"
+
+namespace MapConverter {
```

Comment:
I like it that these are just functions  that do something to a map and return the result. Ideally, could we do something similar for the AllLines Map?